mopidy/ext.py

```diff
@@ -213,23 +209,22 @@ def load_extensions() -> list[ExtensionData]:
 
     :returns: list of installed extensions
     """
-
     installed_extensions = []
 
     for entry_point in metadata.entry_points(group="mopidy.ext"):
         logger.debug("Loading entry point: %s", entry_point)
         try:
             extension_class = entry_point.load()
-        except Exception as exc:
-            logger.exception(f"Failed to load extension {entry_point.name}: {exc}")
+        except Exception:
+            logger.exception(f"Failed to load extension {entry_point.name}.")
             continue
 
         try:
             if not issubclass(extension_class, Extension):
-                raise TypeError  # issubclass raises TypeError on non-class
+                raise TypeError  # noqa: TRY301
         except TypeError:
             logger.error(
-                "Entry point %s did not contain a valid extension" "class: %r",
+                "Entry point %s did not contain a valid extensionclass: %r",
```

Comment:
Did you want one word here?

Reply:
:raised_hands: Fixed in 692cdaa6bfda262c4fc3999012652b40f2b98652 now.